README.md

```diff
@@ -51,8 +54,17 @@ Building depends on:
 
 Fuzzing depends on the theft property-based testing library:
 
- * https://github.com/silentbicycle/theft  
+ * https://github.com/silentbicycle/theft
```

Comment:
The double space here (it means `<br/>`) was intentional!

Reply:
Today I learned!  I thought I had left some garbage there in my editor :-P